### ISSUE TITLE
Should we create another repo to keep track of hardware planning and issues?

### ISSUE THREAD

**Issue by postfalk** [tnc-ca-geo/animl-lambda#5]
I guess if we track all the little bits here it gets crowded and confusing. But I  would like to use Github to keep track of things. I have such a chaos of emails already. I will print them out and put them into a binder with notes to have them ready in the field.

**Comment by nathanielrindlaub** [comment]
Yeah, I like that idea. Does Gary have a github account?

**Comment by postfalk** [comment]
Yes, Gary does have a Github account. It is not exactly his work style and he treats issues as something expensive that has to be sparsely created and not to be destroyed. But he generally respond to questions. His handle is edisiENt
